### DISCUSSION TITLE
Apply controls customization to free drawing

### DISCUSSION THREAD

**Discussion by mutheusalmeida** [fabricjs/fabric.js#8655] [Q&A]
Hey, guys. Is there a way to apply controls customization to the objects created with free drawing? I couldn't figure it out how to do it. Thanks in advance.

**Comment by asturur** [comment]
hi @mutheus , if you get an hold to the object you created, once you have it in a variable, you can change its controls property with something else.

If you describe better what are you trying to do, you can get better help
